crt_portal/static/css/styles.css

```diff
@@ -0,0 +1,3 @@
ublicSans-Bold.ttf) format("truetype")}@font-face{font-family:Public Sans Web;font-style:italic;font-weight:300;font-display:fallback;src:url(../../fonts/public-sans/PublicSans-LightItalic.woff2) format("woff2"),url(../../fonts/public-sans/PublicSans-LightItalic.woff) format("woff"),url(../../fonts/public-sans/PublicSans-LightItalic.ttf) format("truetype")}@font-face{font-family:Public Sans Web;font-style:italic;font-weight:400;font-display:fallback;src:url(../../fonts/public-sans/PublicSans-Italic.woff2) format("woff2"),url(../../fonts/public-sans/PublicSans-Italic.woff) format("woff"),url(../../fonts/public-sans/PublicSans-Italic.ttf) format("truetype")}@font-face{font-family:Public Sans Web;font-style:italic;font-weight:700;font-display:fallback;src:url(../../fonts/public-sans/PublicSans-BoldItalic.woff2) format("woff2"),url(../../fonts/public-sans/PublicSans-BoldItalic.woff) format("woff"),url(../../fonts/public-sans/PublicSans-BoldItalic.ttf) format("truetype")}@font-face{font-family:Merriweather Web;font-style:normal;font-weight:300;font-display:fallback;src:url(../../fonts/merriweather/Latin-Merriweather-Light.woff2) format("woff2"),url(../../fonts/merriweather/Latin-Merriweather-Light.woff) format("woff"),url(../../fonts/merriweather/Latin-Merriweather-Light.ttf) format("truetype")}@font-face{font-family:Merriweather Web;font-style:normal;font-weight:400;font-display:fallback;src:url(../../fonts/merriweather/Latin-Merriweather-Regular.woff2) format("woff2"),url(../../fonts/merriweather/Latin-Merriweather-Regular.woff) format("woff"),url(../../fonts/merriweather/Latin-Merriweather-Regular.ttf) format("truetype")}@font-face{font-family:Merriweather Web;font-style:normal;font-weight:700;font-display:fallback;src:url(../../fonts/merriweather/Latin-Merriweather-Bold.woff2) format("woff2"),url(../../fonts/merriweather/Latin-Merriweather-Bold.woff) format("woff"),url(../../fonts/merriweather/Latin-Merriweather-Bold.ttf) format("truetype")}@font-face{font-family:Merriweather Web;font-style:italic;font-weight:300;font-display:fallback;src:url(../../fonts/merriweather/Latin-Merriweather-LightItalic.woff2) format("woff2"),url(../../fonts/merriweather/Latin-Merriweather-LightItalic.woff) format("woff"),url(../../fonts/merriweather/Latin-Merriweather-LightItalic.ttf) format("truetype")}@font-face{font-family:Merriweather Web;font-style:italic;font-weight:400;font-display:fallback;src:url(../../fonts/merriweather/Latin-Merriweather-Italic.woff2) format("woff2"),url(../../fonts/merriweather/Latin-Merriweather-Italic.woff) format("woff"),url(../../fonts/merriweather/Latin-Merriweather-Italic.ttf) format("truetype")}@font-face{font-family:Merriweather Web;font-style:italic;font-weight:700;font-display:fallback;src:url(../../fonts/merriweather/Latin-Merriweather-BoldItalic.woff2) format("woff2"),url(../../fonts/merriweather/Latin-Merriweather-BoldItalic.woff) format("woff"),url(../../fonts/merriweather/Latin-Merriweather-BoldItalic.ttf) format("truetype")}
```

Comment:
styles.css and styles.css.map used to live in the /compiled folder that was not in source control. 

My preference would be not having these in source control because they are already being compiled on all builds and in my opinion, resolving merge conflicts in compiled files is more trouble than it is worth. Maybe you have some good thoughts on this that I could learn from?